src/core/issues-notes.js

```diff
@@ -133,7 +133,29 @@ function handleIssues(ins, ghIssues, conf) {
     }
     pub(report.type, report);
   });
-  createIssueSummarySection(l10n.issue_summary, issueList);
+  const issueSummaryElement = document.getElementById("issue-summary");
```

Comment:
Can you help me understand this change a bit? I'm not sure why inline this is different/better than the original code?

Reply:
Oh, sorry, I got confused... I was looking at a subset of changes. However, having this in a function makes sense. This code block is getting rather long.